test/base/__init__.py

```diff
@@ -15,6 +15,12 @@
     log.addHandler(logging.StreamHandler())
 
 
+TEST_ROOT_URL = 'https://tc-tests.localhost'
```

Comment:
I see that this URL is only used to test parsing and isn't resolved.  Since that's the case, could we instead make it `https://tc-tests.example.com`?  Subdomains of localhost work out of the box on linux, but on OS X they don't resolve.  We have a lot of contributors to this library who use OS X.  My concern is that this variable might be used to do tests against localhost in new tests and introduce variability between OS X and Linux.  Using a subdomain of example.com will work the same everywhere, so I think we should consider using it.

Reply:
It's the testRootUrl from tc-lib-urls, which is conveniently recognizable as such.  But, it doesn't really matter what it is.  I'll change it.

Reply:
we probably should consider changing it there as well, since it has different behaviour based on the OS that the tests run on.

Reply:
https://github.com/taskcluster/taskcluster-lib-urls/pull/16